README.md

```diff
@@ -1 +1,5 @@
-# DotNetSample
\ No newline at end of file
+# DotNetSample
+
+# will be creating a new project in this repo for testing purposes
+
+# adding another for committing with new user name
```

Comment:
please change this line to only 2 words.

Reply:
yes, it is updated.